herddb-core/src/main/java/herddb/security/sasl/SaslNettyServer.java

```diff
@@ -77,11 +78,24 @@ public SaslNettyServer(Server server, String mech) throws IOException {
 
     }
 
+    private CallbackHandler buildCallbackHandler(String mech) throws IOException {
+        switch (mech + "") {
+            case "MD5-DIGEST":
```

Comment:
Many tests are failing. This should be "DIGEST-MD5".

We should use the constants in SaslUtils for the auth methods